docs/products/flink/howto/create-integration.md

```diff
@@ -3,34 +3,47 @@ title: Create Apache Flink® data service integrations
+With Aiven for Apache Flink®, you can create streaming data pipelines to connect various services.
 
-To use Aiven for Apache Kafka, Aiven for PostgreSQL, or Aiven for
-OpenSearch as a source or target for a Flink application, you will need
-to integrate the related service with Aiven for Apache Flink.
+Aiven for Apache Flink® enables integration with internal and external services,
+such as Aiven managed services, Google BigQuery®, and external Apache Kafka and
+PostgreSQL instances. This flexibility allows you to build data processing pipelines
+using a wide range of data sources and sinks.
 
 ## Create data service integration
 
-You can create Aiven for Apache Flink® data service integrations
-via the [Aiven Console](https://console.aiven.io/) by following these
-steps:
-
-1.  Go to the Aiven for Apache Flink® service page.
-
-2.  If you are setting up the first integration for the selected Aiven
-    for Apache Flink service, click **Get Started** in the service
-    **Overview** screen.
-
-    ![Image of the Aiven for Apache Flink Overview page with focus on the Get Started Icon](/images/content/products/flink/integrations-get-started.png)
-
-3.  To configure the data flow with Apache Flink®, click the Aiven for
-    Apache Kafka®, Aiven for PostgreSQL®, or Aiven for OpenSearch®
-    service that you wish to integrate. Click  <ConsoleLabel name="integrations"/>.
-    to complete the integration process.
-
-    ![Image of the Aiven for Apache Flink Integration page showing an Aiven for Apache Kafka® and an Aiven for PostgreSQL® services](/images/content/products/flink/integrations-select-services.png)
-
-4.  You can include additional integrations by clicking <ConsoleLabel name="plus"/>
-    in the **Data Flow** section.
-
-    ![Image of the Aiven for Apache Flink Integration page showing an existing Aiven for Apache Kafka integration and the + icon to add additional integrations](/images/content/products/flink/integrations-add.png)
+Create Aiven for Apache Flink® data service integrations using the
+[Aiven Console](https://console.aiven.io/). Follow these steps to integrate
+either internal and external services.
+
+### Integrate internal services
+
+1. Log in to [Aiven Console](https://console.aiven.io) and access your
+   Aiven for Apache Flink service.
+1. If this is your first integration for the selected Aiven for Apache Flink service,
+    on the <ConsoleLabel name="overview"/> page, scroll to **Data pipeline**.
+1. Click **Add data source** to initiate the integration setup.
+1. On the **Data service integrations** page, under **Create service integration** tab,
+   choose the Aiven service to integrate: Aiven for Apache Kafka®,
+   Aiven for PostgreSQL®, or Aiven for OpenSearch®.
+1. Click **Integrate**.
+1. To include additional integrations, click <ConsoleLabel name="plus"/> in the
```

Comment:
There's no label for the plus icon, judging from my test so we should use the ConsoleIcon component instead.

Reply:
Good point. Updated this to ConsoleIcon.